ArduCopter/land_detector.cpp

```diff
@@ -65,7 +65,10 @@ void Copter::update_land_detector()
         // because multi's use throttle), check that collective pitch is below land min collective position or throttle stick is zero.
         // Including the throttle zero check will ensure the conditions where stabilize stick zero position was not below collective min. For modes
         // that use altitude hold, check that the pilot is commanding a descent and collective is at min allowed for altitude hold modes.
-        bool motor_at_lower_limit = ((flightmode->has_manual_throttle() && (motors->get_below_land_min_coll() || heli_flags.coll_stk_low) &&
+
+        // check if landing
+        const bool landing = flightmode->is_landing();
+        bool motor_at_lower_limit = ((flightmode->has_manual_throttle() && ((!force_flying || landing) && motors->get_below_land_min_coll() || heli_flags.coll_stk_low) &&
                                     fabsf(ahrs.get_roll()) < M_PI/2.0f) || (motors->limit.throttle_lower && pos_control->get_vel_desired_cms().z < 0.0f));
```

Comment:
```suggestion
                                    fabsf(ahrs.get_roll()) < M_PI/2.0f) || ((!force_flying || landing) && motors->limit.throttle_lower && pos_control->get_vel_desired_cms().z < 0.0f));
```
it belongs on this side of the larger "or" statement because this side of the "or" statement addresses non-manual throttle modes.

Reply:
Oh crap. Sorry.